qiita_ware/ebi.py

```diff
@@ -105,21 +86,44 @@ class EBISubmission(object):
     Parameters
     ----------
     preprocessed_data_id : str
+
+    Raises
+    ------
+    EBISumbissionError
+        - If the action is not in EBISubmission.valid_ebi_actions
+        - If the preprocesssed submitted_to_insdc_status is in
+        EBISubmission.valid_ebi_submission_states
+        - If the prep template investigation type is not in the
+        ena_ontology.terms or not in the ena_ontology.user_defined_terms
+        - If the submission is missing required EBI fields either in the sample
+        or prep template
+        - If the sample preparation metadata doesn't have a platform field or
+        it isn't a EB.Submission.valid_platforms
     """
 
     valid_ebi_actions = ('ADD', 'VALIDATE', 'MODIFY')
     valid_ebi_submission_states = ('submitting', 'success')
+    valid_platforms = ['LS454', 'ILLUMINA', 'UNKNOWN']
+    xmlns_xsi = "http://www.w3.org/2001/XMLSchema-instance"
+    xsi_noNSL = "ftp://ftp.sra.ebi.ac.uk/meta/xsd/sra_1_3/SRA.%s.xsd"
+    experiment_library_fields = [
+        'library_strategy', 'library_source', 'library_selection',
+        'library_layout']
 
     def __init__(self, preprocessed_data_id, action):
         valid_ebi_actions = EBISubmission.valid_ebi_actions
```

Comment:
`self.` only adds 5 letters, I will suggest changing it as it is reducing the number of variables (I had to scroll up to see where the `valid_ebi_actions` was coming from as I expected to see just `self.valid_ebi_actions` - so I needed to check what is different).

These type of variable aliasing makes the code maintenance a bit harder.